src/nativescript-angular/router/page-router-outlet.ts

```diff
@@ -64,18 +64,20 @@ class RefCache {
         </DetachedContainer>`
 })
 export class PageRouterOutlet extends RouterOutlet {
+    // TODO: Figure this out... Help?
+    // @ViewChild('loader', { read: ViewContainerRef }) loaderViewRef: ViewContainerRef;
```

Comment:
The fix is related to this. It may be possible to make `PageRouterOutlet` just a directive now, not sure?